rhdm70-image-streams.yaml

```diff
@@ -22,17 +24,16 @@ items:
         tags: rhdm,xpaas
         supports: rhdm:7.0,xpaas:1.4
         version: '1.0'
-      from:
```

Comment:
This format is correct, see https://github.com/jboss-openshift/application-templates/blob/master/jboss-image-streams.json#L40

Reply:
For some reason, it doesn't work in my testing on the ER1.
If I used the format without adding "dockerImageRepository", it doesn't work.

There is an installation instruction in: https://docs.google.com/document/d/1wmYGz9EJAXCt5gmKLnyOZff1NlvIZv-k4Hc1VycASH8/edit

Reply:
Ok, I’ll take a look on it tomorrow morning.